src/hooks/use-ai-icon.ts

```diff
@@ -0,0 +1,51 @@
+import riveWASMResource from '@rive-app/canvas/rive.wasm';
```

Comment:
Rive usually loads the latest version of the runtime from `unpkg.com`, but as we want Studio to be super fast and work offline, I’m using the packaged version instead.

The assistant isn’t available when offline, but we may want to use this animation runtime in other features that are.